lib/s3-helper.ts

```diff
@@ -0,0 +1,13 @@
+export class S3Helper implements IS3Service {
```

Comment:
IS3Service => IS3Helper, I guess

Reply:
![you-are-right](https://media.giphy.com/media/gmnPIrsAeTBlK/giphy.gif)